{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -30,14 +30,16 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
         If the latter, this application will build all artifacts in
         parallel.
     output_dir
-        The path where the artifact files will be built.
+        The path where the artifact files will be built. The directory
+        will be created if it doesn't exist
     append
         Whether we should append to existing artifacts at the given output
         directory.  Has no effect if artifacts are not found.
     verbose
         How noisy the logger should be.
     """
     output_dir = Path(output_dir)
+    output_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
We should use `vivarium_cluster_tools.mkdir` rather than `mkdir` so that we set the correct `umask`